coordinate/client_test.go

```diff
@@ -64,6 +65,60 @@ func TestClient_Update(t *testing.T) {
 	verifyEqualFloats(t, c.Vec[2], 99.0)
 }
 
+func TestClient_InvalidInPingValues(t *testing.T) {
+	config := DefaultConfig()
+	config.Dimensionality = 3
+
+	client, err := NewClient(config)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Make sure the Euclidean part of our coordinate is what we expect.
```

Comment:
This upper part doesn't seem necessary since we test the coordinate client elsewhere. I'd just try to update with a negative and a too large value and make sure you get the expected error back.